components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -176,18 +178,18 @@ fn migrate_from_sqlcipher_db(
     encryption_key: &str,
 ) -> Result<MigrationMetrics> {
     // encrypt the username/password data
-    let encdec = EncryptorDecryptor::new(encryption_key)?;
+    let enc_dec = EncryptorDecryptor::new(encryption_key)?;
```

Comment:
I think you should probably leave this as `encdec` - not that I have a strong preference for it, but literally every other instance of an `EncryptorDecryptor` is named `encdec` and there are currently zero named `enc_dec`. That will make this patch far far smaller and easier to read.

Reply:
good call -- updated!